lnwallet/chainfee/estimator.go

```diff
@@ -397,6 +402,11 @@ func (b *BitcoindEstimator) EstimateFeePerKW(numBlocks uint32) (SatPerKWeight, e
 //
 // NOTE: This method is part of the Estimator interface.
 func (b *BitcoindEstimator) RelayFeePerKW() SatPerKWeight {
+	err := b.getAndSetMinRelayFee()
```

Comment:
any races we need to worry about for read/write from `b.minFeePerKW`? would imagine not since estimator is single threaded